MaterialSkin/HTML/EN/html/material-skin/index.html.zh_tw

```diff
@@ -0,0 +1,104 @@
+  </ul>
+您可以透過輕觸底部視窗切換或是在螢幕中間左右滑動來切換視窗。如果直接輕觸底部切換欄中當前視窗的按鈕將會有以下功能：
+  <ul>
+    <li瀏覽視窗中 – 會返回上一層，與輕觸返回鍵有一樣的效果。</li>
+    <li>播放視窗中 – 如果 LMS 有安裝插件 'Music and Artist Information' 則會顯示演出者、專輯和曲目資訊。</li>
+    <li>佇列視窗中 – 會滾動至目前播放的曲目。</li>
+  </ul>
+</p>
+<br/>
+
+<h1>桌上型裝置佈局</h1>
+<p>此佈局是專為桌上型裝置如個人電腦或是筆記型電腦所設計的，將畫面在左側放置瀏覽視窗，右側放置佇列視窗，如果有正在播放的歌曲則會將相關資訊顯示
+在底部。透過輕觸主工具列的’展開目前播放’按鈕會將詳細的曲目資訊和大型的封面圖片顯示在主視窗中，同樣地，如果安裝了
+'Music and Artist Information' 插件，輕觸’顯示目前播放曲目’按鈕則會在畫面顯示包含演出者、專輯和歌詞等的詳細資訊。
+</p>
+<br/>
+
+<h1>長觸功能</h1>
+<p>在此應用程式中提供多個長按功能，透過輕觸元素0.5秒或更久來實現。目前提供的長觸功能如下：
+  <ul>
+    <li>於移動裝置畫面底部的切換欄 - 如果應用程式處於瀏覽視窗則長觸瀏覽視窗切換鈕則會返回首頁。</li>
+    <li>長觸瀏覽視窗的返回鍵 - 返回首頁。</li>
+    <li>長觸對話視窗標題（如播放器設定） - 關閉所有對話視窗。</li>
+    <li>長觸音量標籤 - 靜音/取消靜音。</li>
+    <li>長觸開始/暫停按鈕 - 顯示'睡眠'對話視窗。</li>
+    <li>長觸'關閉重覆'圖示或 DSTM（無限符號） - 顯示 DSTM （Don't Stop the Music）選項。</li>
+    <li>長觸佇列中的長度標籤 - 如果有兩個以上播放器時顯示'轉移佇列'對話視窗。</li>
+    <li>長觸工具列播放器名稱旁的播放器圖示 - 如果有兩個以上播放器時顯示'同步'對話視窗。</li>
+    <li>長觸'管理播放器'的選單按鈕 - 如果有兩個以上播放器時顯示'同步'對話視窗。</li>
+    <li>長觸'上ㄧ首'按鈕 - 倒轉 X 秒，如持續觸碰則會連續跳過。跳過的秒數 X 可以額外進行設定。</li>
+    <li>長觸'下一首'按鈕 - 快轉 X 秒，如持續觸碰則會連續跳過。跳過的秒數 X 可以額外進行設定。</li>
+    <li>如果您將插件的個別設定新增到'播放器設定'中，則長觸會將該項新增到首頁。</li>
+  </ul>
+</p>
+<br/>
+
+<h1>管理播放器</h1>
+<p>如果系統中有多個播放器或有多個 LMS 伺服器，於工具列點擊播放器名稱時將會出現'管理播放器'的選單。除了以透過此對話視窗查看所有播放器狀態，如顯示
+顯示播放曲目和音量等，也可以進行以下操作：
+  <ul>
+    <li>列出正常播放器和群組播放器，如果您在 LMS 安裝'群組播放器'插件後才會顯示群組播放器</li>
+    <li>同步播放器。透過將播放器拖曳到另一播放器身上時將會建立同步的群組播放器。如果要將其刪除則可以將播放器拖曳到主工具列中（將會顯示拖曳到
+此處來脫離同步群組）。</li>
+    <li>如果在 LMS 中有安裝'群組播放器'插件，則您將允許透過此對話視窗來建立、編輯或刪除群組。位於視窗左上角的選單內有建立群組的功能。</li>
+    <li>如果在區域網路內有多個 LMS 伺服器，此對話視窗可用於將播放器移動到不同的伺服器之間。</li>
+    <li>設定為預設播放器。透過將播放器設為預設，當 Material 重新啟動後將會自動選取該播放器。如果在重新啟動後預設播放器不可見，那麼將會選取順位
+    中的下一個播放器，但是如果預設播放器之後再度出現，將會重新選取預設播放器。透過位於播放器最右側的選單按鈕（三點圖示）來將播放器設為預設。</li>
+  </ul>
+目前所使用的播放器將會以半透明形式來突出顯示，而在預設播放器旁則會有勾號進行提示。任何啟用了睡眠計時器的播放器會在名稱旁顯示床的圖案。
+</p>
+<br/>
+
+<h1>設定</h1>
+<p>由於 LMS 的性質，會出現多個對話視窗。
+  <ul>
+    <li>界面 - 包含控制 Material Skin 的佈局，此處的任何設定都將儲存在瀏覽器中，因此每個設定都會音使用者而異。可以透過位於對話視窗右上角中的
+    選單中的'存為預設'和'恢復預設'選項來變更預設值。變更伺服器設定中的 Material Skin 設定將會影響所有使用者。</li>
+    <li>播放器 - 包含個別播放器的設定，例如名稱、鬧鐘等。僅管 LMS 有許多設定，但是在預設情況下只會顯示依定數量的設定內容。如要瀏覽更多播放器設
+    定，可以透過輕觸對話視窗底部的'更多設定'來查看。如果您的播放器為 piCorePlayer 或 SqueezeESP32，則該視窗將會提供讓您訪問基於網頁的裝置設
+    定頁面的按鈕。</li>
+    <li>伺服器 - 包含所有 LMS 伺服器設定，例如安裝插件等。</li>
+  </ul>
+</p>
+<br/>
+
+<h1>瀏覽模式</h1>
+<p>ㄧ般來說瀏覽模式的清單設定（如演出者、專輯和音樂風格）是因播放器而異的，這就是為什麼在'播放器設定'的'額外設定'中存在一個'其他瀏覽模式'的原因。
+然而，由於此應用程式旨在控制多個播放器，因此該可用的瀏覽模式清單是在'設定 -> 界面'中設定的。在該對話視窗中可以看見一組複選框，用於控制顯示在首頁
+顯示的項目。'我的音樂'選項旁有個齒輪圖示，按下此圖示將出現一個對話視窗來設定此應用程式要顯示的瀏覽模式。
+</p>
+<br/>
+
+<h1>虛擬媒體庫</h1>
+<p>LMS 中的虛擬媒體庫是建立子音樂收藏庫的一種方式，例如您能夠建立專屬於每位家庭成員的虛擬媒體庫
+</p>
+<p>LMS 會為每個播放器的活動庫設定瀏覽模式，透過此應用程式，您可以在'播放器設定'中為播放器設定庫別。此媒體庫會將 LMS 的特定功能（如混合播放）限制
+使用在所選的曲目，然而，此應用程式與 LMS 不同之處在於，透過此應用程式所使用的建立的媒體庫，在不同的播放器所看到的內容會一致。也就是說，某位家庭成
+員無論控制哪個播放器都可以查看到自己想要媒體庫。
+</p>
+<p>但注意，在您實際建立這些虛擬庫之前是不可見也不可選。有多種方法可以做到這一點；透過儲存'進階搜尋'（點擊首頁中'搜尋'選項內的出現的'進階搜尋'選項）
+或使用'Simple library views'等插件來實現。
```

Comment:
Should this not also be translated?

Reply:
It's plugin related, should be translate later after the slimserver is translated.